chromium_src/services/service_manager/sandbox/win/sandbox_win.cc

```diff
@@ -0,0 +1,13 @@
+#include "base/command_line.h"
+#include "base/process/launch.h"
+
+namespace {
+void BraveLaunchOption(base::CommandLine* cmd_line,
+                       base::LaunchOptions *options) {
+  // tor::swtiches::kTorExecutablePath
```

Comment:
so why not just use `tor::swtiches::kTorExecutablePath` like the comment says?

Reply:
that will have to patch the BUILD.gn and also leads to circular deps.
Probably can do when we migrate to components/tor